doc/index.rst

```diff
@@ -43,99 +87,81 @@ automatic differentiation, and optimization of hybrid quantum-classical computat
 
 
 - *Follow the gradient*.
-  Built-in **automatic differentiation** of quantum circuits
+  Built-in **automatic differentiation** of quantum circuits.
 
 ..
 
 - *Best of both worlds*.
-  Support for **hybrid quantum and classical** models
+  Support for **hybrid quantum and classical** models.
 
 ..
 
 - *Batteries included*.
-  Provides **optimization and machine learning** tools
+  Provides **optimization and machine learning** tools.
 
 ..
 
 - *Device independent*.
-  The same quantum circuit model can be **run on different backends**
+  The same quantum circuit model can be **run on different backends**. Install
+  `plugins <https://pennylane.ai/plugins.html>`_ to run your computational circuits
+  on even more devices, including **Strawberry Fields**, **Rigetti Forest**, **ProjectQ**,
+  **Microsoft QDK**, and **IBM Q**.
 
 ..
 
 - *Compatible with existing machine learning libraries*.
```

Comment:
header doesn't fit with the others, it's too long

Reply:
Any suggestions? How about "Connects with PyTorch and TensorFlow"